scripts/checkpatch_inc.sh

```diff
@@ -7,9 +7,14 @@ CHECKPATCH_IGNORE=$(echo core/lib/lib{fdt,tomcrypt} lib/lib{png,utils,zlib} \
 _CP_EXCL=$(for p in $CHECKPATCH_IGNORE; do echo ":(exclude)$p" ; done)
 
 function _checkpatch() {
+		# Use typedefsfile if supported by the checkpatch scripts
+		a="--typedefsfile typedefs.checkpatch"
```

Comment:
No, but since it's working we should not bother I suppose. Well, one better way may be `checkpatch.pl --help 2>&1 | grep -q typedefsfile`.